.github/workflows/auto-instrumentation.yml

```diff
@@ -21,7 +21,7 @@ env:
   PYTHON_VERSION: '3.11'
   PIP_VERSION: '22.0.4'
   REQUIREMENTS_PATH: "internal/buildscripts/packaging/tests/requirements.txt"
-  GO_VERSION: 1.20.10
+  GO_VERSION: ~1.21.5
```

Comment:
The `~` addition is not required here, it just allows more flexibility with which version of `go` is used, and now matches `contrib` and `core` more closely. I'm happy to revert if that's preferred.

Reply:
I'm actually going to remove this for now, if we want it we can just open another PR.